packages/font-family-lang-support/readme.md

```diff
@@ -0,0 +1,439 @@
+
+# @creately/font-family-lang-support
+
+> This package will be useful to check whether the given language is supported by the given font family. Returns true if it is supported.
+
+## Install
+
+```
+$ npm install @creately/font-family-lang-support
+```
+
+## Usage
+
+```js
+import { FontLanguageInfo } from '@creately/font-family-lang-support';
+
+const fontLanguageInfo = new FontLanguageInfo()
+                                .isFontSupport(fontFamily: string, languageCode: string)
+
+```
+
```

Comment:
@sajeevah since we are expecting the font family in a certain ways, can you please add the sample input and output format? and also like you added the languages list, add the supported font family lists as well.